lib/sprockets/uglifier_compressor.rb

```diff
@@ -47,14 +47,24 @@ def call(input)
       if Autoload::Uglifier::VERSION.to_i < 2
```

Comment:
Given this condition is already here, it seems simpler to combine them into a `case`.